packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -738,19 +728,21 @@ export const audiusBackend = ({
     sdk: AudiusSdk
   }): Promise<BN | null> {
     try {
-      const { userBank } =
-        await sdk.services.claimableTokensClient.getOrCreateUserBank({
-          ethWallet: ethAddress,
-          mint: 'wAUDIO'
-        })
+      const userBank = await sdk.services.claimableTokensClient.deriveUserBank({
```

Comment:
We accidentally introduced a bug that caused user banks to get created when checking balances - this undoes that